CatoNetworks/cato-sase/tests/cato_sase_ng_antimalware_events.json

```diff
@@ -0,0 +1,73 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "cato-sase",
+        "dialect_uuid": "469bd3ae-61c9-4c39-9703-7452882e70da"
+      }
+    },
+    "message": "{\"account_id\":\"1714\",\"action\":\"Block\",\"application\":\"Suspected apps\",\"dest_country\":\"Ireland\",\"dest_ip\":\"52.51.102.52\",\"dest_port\":\"443\",\"domain_name\":\"reflector.peterljames.org\",\"event_count\":\"1\",\"event_sub_type\":\"NG Anti Malware\",\"event_type\":\"Security\",\"file_hash\":\"70355dcf91019652e32eba67140a2708232a1fa786f90446d7984afe314f63f3\",\"file_name\":\"eicar.exe\",\"file_size\":\"68\",\"indicator\":\"EICAR-SENTINEL-ANTIVIRUS-TEST-FILE\",\"internalId\":\"QCzt6ht6GY\",\"os_type\":\"OS_MAC\",\"os_version\":\"11.6.0\",\"pop_name\":\"London\",\"rule\":\"0\",\"rule_id\":\"0\",\"src_ip\":\"10.41.173.156\",\"src_is_site_or_vpn\":\"VPN User\",\"src_site\":\"Peter James\",\"threat_name\":\"malware\",\"threat_verdict\":\"virus_found\",\"time\":\"1651045480623\",\"url\":\"https://reflec.xxx.com /eicar.exe\",\"vpn_user_email\":\"peter@xxx.com\",\"event_timestamp\":\"2022-04-27T07:44:40Z\"}"
+  },
+  "expected": {
+    "message": "{\"account_id\":\"1714\",\"action\":\"Block\",\"application\":\"Suspected apps\",\"dest_country\":\"Ireland\",\"dest_ip\":\"52.51.102.52\",\"dest_port\":\"443\",\"domain_name\":\"reflector.peterljames.org\",\"event_count\":\"1\",\"event_sub_type\":\"NG Anti Malware\",\"event_type\":\"Security\",\"file_hash\":\"70355dcf91019652e32eba67140a2708232a1fa786f90446d7984afe314f63f3\",\"file_name\":\"eicar.exe\",\"file_size\":\"68\",\"indicator\":\"EICAR-SENTINEL-ANTIVIRUS-TEST-FILE\",\"internalId\":\"QCzt6ht6GY\",\"os_type\":\"OS_MAC\",\"os_version\":\"11.6.0\",\"pop_name\":\"London\",\"rule\":\"0\",\"rule_id\":\"0\",\"src_ip\":\"10.41.173.156\",\"src_is_site_or_vpn\":\"VPN User\",\"src_site\":\"Peter James\",\"threat_name\":\"malware\",\"threat_verdict\":\"virus_found\",\"time\":\"1651045480623\",\"url\":\"https://reflec.xxx.com /eicar.exe\",\"vpn_user_email\":\"peter@xxx.com\",\"event_timestamp\":\"2022-04-27T07:44:40Z\"}",
+    "event": {
+      "kind": "event",
+      "action": "Block"
```

Comment:
```suggestion
      "action": "block",
      "category": ["malware"],
      "type": ["info"]
```

Reply:
I think it might be threat?
https://www.elastic.co/guide/en/ecs/8.7/ecs-allowed-values-event-category.html#ecs-event-category-threat
https://support.catonetworks.com/hc/en-us/articles/360011568478

Reply:
Because we have a threat name and type here

Reply:
For me, this event represents a malware detection. The `event.category` `Threat` is for threat intelligence, when collecting indicators of compromise (IoC), with which we are rarely confronted